api/v1/helpers/verbs/doPost.js

```diff
@@ -97,6 +96,10 @@ function doPost(req, res, next, props) {
   }
 
   return postPromise.then((o) => {
+    if (!o) { // could be here from u.handleError
+      return;
```

Comment:
does this mean the user is getting back the error properly now? where is the test to show this?